lib/ansible/modules/cloud/amazon/elasticache_parameter_group.py

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+try:
```

Comment:
flake8 will moan about this - and apologies for not being clearer. Leave this section where it was, and then everything should be fine. One thing to note while I'm in the area,but it's not a blocker, is that flake8 will complain less if you add HAS_BOTO3 to the list of imports and don't import boto3 (in the `except ImportError` part, just `pass`, as you'll handle that  using HAS_BOTO3 later)